### PR TITLE
[SYCL][L0] Bump minimum shared read-only allocation slab to 2M

### PR DESCRIPTION
**Pull request by smaslov-intel**
Signed-off-by: Sergey V Maslov <sergey.v.maslov@intel.com>

### CODE REVIEW

File: sycl/plugins/level_zero/usm_allocator.cpp [hunk 1]
```diff
@@ -72,46 +65,63 @@ static sycl::detail::SpinLock PoolLock;
...
+    Capacity[MemType::Shared] = 0;
+    SlabMinSize[MemType::Shared] = 2_MB;
 
     // Allow pooling of shared allocations that are only modified on host.
-    MaxPoolableSize[SystemMemory::SharedReadOnly] = 4_MB;
-    Capacity[SystemMemory::SharedReadOnly] = 4;
-    SlabMinSize[SystemMemory::SharedReadOnly] = 64_KB;
+    MaxPoolableSize[MemType::SharedReadOnly] = 4_MB;
+    Capacity[MemType::SharedReadOnly] = 4;
+    SlabMinSize[MemType::SharedReadOnly] = 2_MB;
```
Comment (smaslov-intel) [inline sycl/plugins/level_zero/usm_allocator.cpp:124]: this is the key change in this PR

**Review by jandres742 — COMMENTED** [review]
+1

**Review by bso-intel — APPROVED** [review]
LGTM

**Comment by pvchupin** [comment]
@smaslov-intel, can you add @bso-intel to @intel/dpcpp-l0-pi-reviewers?

**Pull request merged**